lib/trading/close_prices.txt

```diff
@@ -0,0 +1 @@
+926.6900024414062,875.280029296875,857.739990234375,919.1300048828125,908.8800048828125,879.4400024414062,878.3699951171875,884.5499877929688,893.97998046875,903.719970703125,914.3499755859375,942.8900146484375,950.02001953125,925.6099853515625,902.5,903.5599975585938,903.6300048828125,894.52001953125,889.6400146484375,859.0499877929688,880.0800170898438,871.3300170898438,853.5399780273438,870.3900146484375,906.1599731445312,881.8599853515625,860.010009765625,874.1500244140625,840.3499755859375,846.7100219726562,762.0
```

Comment:
Nice work
can you please explain what the close_prices are and why you chose to save them in a file on the server... 😕

Reply:
The close prices are the last prices traded on a specified range of trading days. The file is created using a python script and then the prices are retrieved by the bot class. It shouldn't be tracked by git though so I've added it to .gitignore

Reply:
Ok

I think that's best